client/src/pages/Background.js

```diff
@@ -0,0 +1,306 @@
+import React, { Component } from "react";
+
+import {
+  Typography,
+  Grid,
+  Paper,
+  Button,
+  Box,
+  TextField,
+  Snackbar,
+  FormControl,
+  Select,
+  InputLabel,
+  MenuItem,
+} from "@material-ui/core";
+import MuiAlert from "@material-ui/lab/Alert";
+import Rating from '@material-ui/lab/Rating';
+import { withStyles } from "@material-ui/core/styles";
+import { Route, Link } from "react-router-dom";
+import pic1 from "../assets/pic1.png";
+
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+const loginStyle = (theme) => ({
+  root: {
+    flexGrow: 1,
+    height: "100vh",
+  },
+  paper: {
+    padding: theme.spacing(0),
+    margin: "auto",
+    width: "Full",
+    height: "100%",
+    border: 0,
+  },
+  image: {
+    //    width: '90vh',
+    height: "100vh",
+  },
+  img: {
+    margin: "auto",
+    display: "block",
+    maxWidth: "100%",
+    maxHeight: "100%",
+  },
+  text: {
+    paddingTop: "0.5rem",
+    paddingLeft: "20%",
+    paddingRight: "20%",
+  },
+  account: {
+    paddingTop: "0.5rem",
+  },
+  button: {
+    borderRadius: 35,
+    marginLeft: "1rem",
```

Comment:
```suggestion
    margin: "auto 1.5rem 2em 1rem",
```

this would save you multiple lines of CSS

Reply:
Great suggestion. I will use one line instead of 4 lines in the project.